components/EmbraceModal.vue

```diff
@@ -1,14 +1,7 @@
 <template>
   <ModalWithBackdrop v-model="showModal">
-    <div class="pt-28 mb-10 max-w-2xl h-128 max-h-full text-slate-800 text-left">
-      <form class="justify-center items-center bg-sushi-100 flex flex-col px-10 rounded-2xl max-md:px-5">
-        <div
-          class="justify-between items-stretch self-stretch flex gap-5 mt-10 max-md:max-w-full max-md:flex-wrap max-md:mt-10"
-        >
-          <div class="text-2xl font-bold grow shrink basis-auto max-md:max-w-full">
-            {{ embracePage?.data.preview_title || 'Email Preview' }}
-          </div>
-        </div>
+    <div class="pt-2 mb-10 max-w-2xl h-128 max-h-full text-slate-800 text-left">
```

Comment:
you can move this styling to the `form` tag below and then remove this div. I don't see the necessity to have another div as wrapper if there is only one child :) I've also tested that locally and it looks the same